projects/plugins/jetpack/_inc/client/recommendations/feature-utils.js

```diff
@@ -83,8 +83,14 @@ export const mapStateToSummaryFeatureProps = ( state, featureSlug ) => {
 	}
 };
 
-export const mapStateToSummaryResourceProps = ( state, resourceSlug ) => {
```

Comment:
Removed state as it was not being used

Reply:
I would also rename the function as well to match the new logic (e.g. `getSummaryResourceProps` or similar)

Reply:
Nice catch! Will do